api/firmware/messages/antiklepto.pb.go

```diff
@@ -15,7 +15,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.34.2
-// 	protoc        v5.27.1
+// 	protoc        v3.21.12
```

Comment:
We are using an older version, which sounds weird, but I guess it doesn't matter as the output is the same.